src/base/ApplicationTrait.php

```diff
@@ -1224,6 +1225,17 @@ public function getLocale(): Locale
         return $this->get('locale');
     }
 
+    /**
+     * Returns the log dispatcher component.
+     *
+     * @return Dispatcher the log dispatcher application component.
+     */
+    public function getLog(): Dispatcher
+    {
+        /** @noinspection PhpIncompatibleReturnTypeInspection */
+        return $this->get('log');
+    }
```

Comment:
This is identical to `yii\base\Application::getLog()` so could be removed.

Reply:
This returns our `Dispatcher`, not Yii's.

Reply:
Oh sorry, I missed the updated return type.